packages/clay-css/src/images/icons/content-shield.svg

```diff
@@ -0,0 +1,11 @@
+<!--
+* SPDX-FileCopyrightText: © 2024 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2024 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1.625V11.375C0 12.2713 0.728394 13 1.625 13H7V11.375H1.625V1.625H11.375V6H13V1.625C13 0.728716 12.2716 0 11.375 0H1.625C0.728394 0 0 0.728716 0 1.625ZM9.75 4.0625C9.75 4.95997 9.0224 5.6875 8.125 5.6875C7.2276 5.6875 6.5 4.95997 6.5 4.0625C6.5 3.16503 7.2276 2.4375 8.125 2.4375C9.0224 2.4375 9.75 3.16503 9.75 4.0625ZM3.25 5.6875L4.875 2.4375L6.5 5.6875H3.25ZM7.00002 9V10H3.50007C2.81257 10 2.84382 9 3.50007 9H7.00002ZM7.00002 8V7.42425C6.999 7.29597 7.03622 7.17011 7.10724 7.06172C7.12131 7.04026 7.13658 7.01966 7.15296 7H3.5C2.84375 7 2.8125 8 3.5 8H7.00002Z"/>
```

Comment:
@SelenaAungst these indent's are using spaces. It should be tabs. Your code editor should have something that can convert to tabs.